### PR TITLE
Style the whole damn app

### LINKED ISSUE #82: Add Materialize library

### PR DESCRIPTION
**Pull request by troym9731**
Thought I'd add this to begin with, but I'm changing my mind frequently, so this will probably end up being a larger ticket.

EDIT: Hah, boy was the statement above an understatement.

Closes https://github.com/az-webdevs/azwebdevs.org/issues/81
Closes https://github.com/az-webdevs/azwebdevs.org/issues/74
Closes https://github.com/az-webdevs/azwebdevs.org/issues/28
Closes https://github.com/az-webdevs/azwebdevs.org/issues/45
Closes https://github.com/az-webdevs/azwebdevs.org/issues/31
Closes #82

### CODE REVIEW

File: web/templates/layout/app.html.eex [hunk 1]
```diff
@@ -17,6 +18,7 @@
       <%= render @view_module, @view_template, assigns %>
     </div>
 
-    <script src="<%= static_path(@conn, "/js/app.js") %>"></script>
```
Comment (therealklanni) [inline web/templates/layout/app.html.eex]: Aww, but I liked our empty js :P

**Comment by therealklanni** [comment]
Mind updating the PR description with 

```
Closes #X
Closes #Y
```

when you get a chance?

**Comment by troym9731** [comment]
Sure. Once I have the base site looking fine, I'll probably split up the rest of the styling into separate PRs. This PR is already out of hand.

**Comment by therealklanni** [comment]
Sounds good!

**Pull request merged**